src/tests/ftest/control/daos_system_query.yaml

```
@@ -0,0 +1,27 @@
+hosts:
+  test_servers: 1
```

Technically, ftest runs `daos` on a client. That currently defaults to localhost, but eventually will not. So we should be explicit here
```suggestion
hosts:
  test_servers: 1
  test_clients: 1
```

---

Actually, ignore this one. We currently always run `daos` commands on localhost